resources/views/events/aanmelden.blade.php

```diff
@@ -41,6 +41,18 @@
                         <div class="alert alert-danger">{{ $message }}</div>
                         @enderror
                     </div>
+                    <div>
+                        <label for="gender" class="col-sm-2 col-form-label">
+                            <span class="requiredStar">*</span>Geslacht
+                        </label>
+                        <div class="col-sm-8">
+                            <input type="text" class="form-control" id="gender"
```

Comment:
Waarom is geslacht een tekstveld? Vgsm is het niet handig als iemand 'blabla' in kan typen. Dropdown lijkt me beter

Reply:
nog een toevoeging met gender ipv geslacht: naar mijn mening zijn de 5 meest gebruikte genders voldoende om genoeg af te dekken. Andere opties kan je afdekken met 'anders'
![image](https://user-images.githubusercontent.com/103929631/233089510-298f3ac4-e792-40d5-a7da-f90ed2e87238.png)